### PR TITLE
OC2MC Stabilization and better TMUX management

### PR DESCRIPTION
**Pull request by fredyvilla**
This PR introduces versions for the docker container to baseline against Spock, Minecraft 1.8.8 and add tmuxinator to improve management of ros nodes.

### CODE REVIEW

**Comment by AmeBel** [comment]
thanks :-)

One thing though, why version the Images? AFAIK, oc2mc isn't versioned, and the docker images are merely for development and not for production environment, what am I not considering?

**Comment by fredyvilla** [comment]
When I started changing the Dockerfile to add tmuxinator I realized that it
would be a good idea to version the images so that we could create
containers for different baselines. I created a tag/version for oc2mc to
1.1 so I figured we could add keep the versions synched.  The version also
freezes versions of our key dependencies with Minecraft and SpockControl.
They keep moving to new versions and breaking our code. Since the goal is
to advance the Bot AGI and not have to keep rewriting the piping, I figured
this strategy would help with that.  It is also a best practice to version
docker images otherwise the latest tag doesn't mean much.

**Comment by AmeBel** [comment]
I just saw the tag on oc2mc, great. But why did you choose 1.1 for start of versioning? Do you think its integration with the rest of the pipeline is stable enough to warrant a version 1.*.*`?

**Comment by fredyvilla** [comment]
It is stable but not very feature rich. We can start at a 0.1. Or use any other versioning scheme you prefer.

**Comment by AmeBel** [comment]
The other repos are guided by http://semver.org/. So, as you suggested, it is best to start with `0.1.0`.

**Comment by fredyvilla** [comment]
Changed the version as requested

File: opencog/docker-build.sh [hunk 1]
```diff
@@ -126,7 +126,7 @@ fi
 if [ $BUILD_EMBODIMENT_IMAGE ] ; then
     check_dev_cli
     echo "---- Staring build of opencog/minecraft ----"
-    docker build $CACHE_OPTION -t opencog/minecraft minecraft
+    docker build $CACHE_OPTION -t opencog/minecraft:1.1 minecraft
```
Comment (AmeBel) [inline opencog/docker-build.sh]: missed one

**Comment by AmeBel** [comment]
Thanks :smile:

**Pull request merged**